config/sky-pages/sky-pages.config.js

```diff
@@ -22,132 +22,137 @@ function readConfig(file) {
   return fs.readJsonSync(file, 'utf8');
 }
 
-module.exports = {
+/**
```

Comment:
There are zero functional changes to this file; I simply adjusted the format so we can use a few of its methods (namely `spaPath`) via destructuring. (See my changes to `common.webpack.config.js` for an example.)

Reply:
Just want to say I appreciate you calling this out.  It definitely helps when reviewing.